### DISCUSSION TITLE
Install Cobbler in Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by sqsous** [cobbler/cobbler#3709] [Q&A] [answered]
Hello, 
I have been searching for a while and can't seem to find an answer
is there a way to install cobbler in ubuntu 20.04?

**Comment by SchoolGuy** [comment, marked as answer]
Yes. Here are the instructions for the community maintained Cobbler 3.3.x release series:

https://software.opensuse.org/download/package?package=cobbler&project=systemsmanagement%3Acobbler%3Arelease33

Just click on "Ubuntu" and then use the instructions for "xUbuntu 20.04".

If you are familiar with the build workflow you can checkout the latest tag and copy and adjust the build workflow for Debian 12 to Ubuntu.

**Reply by sqsous** [reply]
Thank you!!
